README.md

```diff
@@ -13,34 +13,22 @@ The coordinate types LV95 as well as the older LV03 are supported. More informat
 Only German and French borehole profiles are supported as of now.
 
 ## Installation
-We use conda to create and manage the project's dependencies. The project comes with two environments, `environment-dev.yml` and `environment-prod.yml`, respectively. The prod environment contains all necessary dependencies to run the code and extraction pipelines therein. All dependencies that are useful for the development of the code, but not to run it, are separated into the dev environment.
-
-Assuming you have conda installed and cloned the repository, run the following command in your project repository:
-```bash
-conda env create -f environment-prod.yml
-```
-
-If you would like to get all developer functionalities, run:
-
-```bash
-conda env create -f environment-dev.yml
-```
+We use pip to manage the packages dependencies. We recommend using a virtual environment within which to install all dependencies.
 
+Local installation for development: `pip install -e .[all]`
+Installation from source for production scenarios: `pip install git+https://github.com/swisstopo/swissgeol-boreholes-dataextraction.git"
 
 ## Run data extraction
 To execute the data extraction pipeline, follow these steps:
 
-1. **Activate the Conda environment**
-
-   If you haven't already, activate the conda environment using the following command:
-
-   ```bash
-   conda activate boreholes-prod
-   ````
-
-    If you are developing and testing the code, you might want to use the dev environment instead:
+1. **Activate the virtual environment**
 
-    `conda activate boreholes-dev`
+    Activate your virtual environment and install the package. On unix systems this is
+    ```bash
+    python -m venv env
+    source env/bin/activate
+    pip install -e .[all]
+    ```
```

Comment:
I think it makes more sense to include these commands under the "installation" section, and here only say: activate the virtual environment if it is not active yet, using `source env/bin/activate`?